scripts/ableplayer-base.js

```diff
@@ -30,6 +30,11 @@
 	// https://developer.mozilla.org/en-US/docs/Web/API/Window/speechSynthesis
 */
 
+// Add dependencies here so that including ableplayer in another
+// package.json, it comes with dependencies included
+var jQuery = require("jquery");
+var Cookies = require("js-cookie");
+
```

Comment:
This covers this file, but not all the others. It works by coincidence because this file ends up being the first one and the build process literally concatenates all the files.

I think it would be safer to add these imports to each of the source files, not just this one.

Reply:
yeh I thought it'd be fine coz it's the first file and everything's concatenated into `ableplayer.dist.js`... but ok I'll add it to each source file just to be safe

Reply:
I say that because if I was to do much more work on this library, step 1 would be removing the antiquated build process that just concatenates files - scope is useful!

Reply:
Ok done - I added `jQuery` to all js files, but only the ones need Cookies are added for `js-cookie` :)